frontend/src/components/form/DatePicker.vue

```diff
@@ -24,7 +24,7 @@ const {
   maxDate?: Date;
 }>();
 
-const { errorMessage, value } = useField<Date>(name);
+const { errorMessage, handleBlur, value } = useField<string>(name);
```

Comment:
Is PrimeVue now handling the date internally as a string?

Reply:
Fixed accidental replace, Date type is back.